llvm/tools/llvm-exegesis/lib/RISCV/Target.cpp

```diff
@@ -103,7 +103,7 @@ bool ExegesisRISCVTarget::checkOpcodeSupported(
   FeatureBitset RequiredFeatures = RISCV_MC::computeRequiredFeatures(Opcode);
   FeatureBitset MissingFeatures =
       (AvailableFeatures & RequiredFeatures) ^ RequiredFeatures;
-  return MissingFeatures.none();
+  return !(MissingFeatures.none());
```

Comment:
I think this was right before. Simplifying the previous bit operation:
```
FeatureBitSet MissingFeatures = RequiredFeatures & ~AvailableFeatures;
// MissingFeatures now has bits set which are in RequiredFeatures, but not in available features.
// If MissingFeatures is none, then the required features are a subset of the available features
return MissingFeatures.none();
```

Reply:
Addressed